hazelcast/serialization/portable/reader.py

```diff
@@ -296,6 +367,46 @@ def _check_factory_and_class(field_def, factory_id, class_id):
         )
 
 
+def read_portable_date(inp: ObjectDataInput):
+    y, m, d = _read_portable_date(inp)
+    return datetime.date(y, m, d)
+
+
+def _read_portable_date(inp: ObjectDataInput):
+    y = inp.read_short()
+    m = inp.read_byte()
+    d = inp.read_byte()
+    return y, m, d
+
+
+def read_portable_time(inp: ObjectDataInput):
+    h, m, s, ms = _read_portable_time(inp)
+    return datetime.time(h, m, s, ms)
+
+
+def _read_portable_time(inp: ObjectDataInput):
+    h = int(inp.read_byte())
+    m = int(inp.read_byte())
+    s = int(inp.read_byte())
+    nanos = int(inp.read_int() / 1000)
```

Comment:
maybe we can use `//` operator instead

Reply:
Fixed at https://github.com/hazelcast/hazelcast-python-client/commit/6b34418b0dd46c23cb7a5ed87e84d83865ce27e1.